src/targets/gpu/target.cpp

```diff
@@ -129,9 +129,11 @@ std::vector<pass> target::get_passes(migraphx::context& gctx, const compile_opti
     unsupported_fp8e4m3fnuz_ops.insert("argmin");
 
     std::set<std::string> unsupported_fp8ocp_ops = {};
-    // TODO update with hipBLASLt support
+    // TODO: remove this when the flag is removed
+#if !MIGRAPHX_ENABLE_HIPBLASLT_GEMM
```

Comment:
Probably need to do `MIGRAPHX_DECLARE_ENV_VAR(MIGRAPHX_ENABLE_HIPBLASLT_GEMM)` at the top of the file...

Reply:
I'm not sure what that does exactly. Is the env variable not declared elsewhere?

Reply:
This seems to be the way MIGraphX handles env vars. Doing this provides functions like enabled/disabled etc to work with the env vars, so we can set any of the following values for the env variable:
```
Set to "1", "enable", "enabled", "yes", or "true" to use.
```

Also, I missed to mention that we should do `if(not enabled(MIGRAPHX_ENABLE_HIPBLASLT_GEMM{}))` instead of `!MIGRAPHX_ENABLE_HIPBLASLT_GEMM`, for guarding using the env variable.

Reply:
Does this environment variable not clash with the one already declared in lowering.cpp? Thanks.

Reply:
I don't think they should clash.